src/gsi/gesinfo.F90

```diff
@@ -255,6 +255,9 @@ subroutine gesinfo
            jcap=gfshead%jcap, ntrac=gfshead%ntrac, idvc=gfshead%idvc, &
            idsl=gfshead%idsl,   ncldt=gfshead%ncldt, iret=iret2)
 
+        ! FV3GFS write component does not include JCAP, infer from DIMY-2
+        !if (gfshead%jcap<0) gfshead%jcap=gfshead%latb-2
```

Comment:
Delete these lines, if they were added but commented out then they are not needed.

Reply:
Deleted.